code/Serving_API.py

```diff
@@ -98,6 +119,7 @@ def build_up_global_graph(db_file):
     submarine_to_standard_paths_pairs = get_all_submarine_to_standard_paths_pairs(
         db_file)
     standard_paths = get_all_standard_paths(db_file)
+    asn_nodes = get_all_asn_node(db_file)
```

Comment:
See below, I'd call this get_all_as_locations. Each node represent a location, which is not an ASN.

Reply:
fix all asn_nodes to all_as_locations while get_all_asn_node to get_all_as_locations